godel_surface_detection/CMakeLists.txt

```diff
@@ -71,6 +71,11 @@ add_executable(surface_blending_service  src/services/surface_blending_service.c
 target_link_libraries(surface_blending_service ${PROJECT_NAME})
 add_dependencies(surface_blending_service godel_msgs_generate_messages_cpp)
 
+## robot_auto_blending
+add_executable(robot_auto_blending  src/nodes/robot_auto_blending.cpp)
```

Comment:
Try avoid using generic commit messages whenever possible.  In general, commit message should be brief but still provide relevant information about the change that it makes.